rfcs/convergence/fixed-versions-for-prerelease.md

```diff
@@ -154,7 +154,22 @@ To mitigate this problem during the prerelease phase we can do the following:
 ### Prefer to release new components straight into react-components beta
 
 For sane versioning, new packages should be released straight into the `beta` prerelease tag. A new way of consuming
-unstable packages is proposed in the next section.
+unstable packages is proposed in the next section. `alpha` and `beta` are liable to breaking API changes both internally
+and externally. Having both prerelease tags existing side by side could cause dependency hell too easily.
+
+### RC release can coexist with beta components
+
+As Fluent v9 moves to the `RC` stage, we want to clearly communicate to partners what they can expect from the stable
```

Comment:
nit, might be good to expand RC to "release candidate" at least once (even though it's a pretty common term)?